diff --git a/node_modules/react-native-reanimated/src/reanimated2/NativeMethods.ts b/node_modules/react-native-reanimated/src/reanimated2/NativeMethods.ts
index b29ac0f..9eb2ad4 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/NativeMethods.ts
+++ b/node_modules/react-native-reanimated/src/reanimated2/NativeMethods.ts
@@ -62,7 +62,7 @@ if (isWeb()) {
       return null;
     }
 
-    const measured = _IS_FABRIC
+    const measured = global._IS_FABRIC
       ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         _measureFabric!(viewTag as ShadowNodeWrapper)
       : // eslint-disable-next-line @typescript-eslint/no-non-null-assertion